### PR TITLE
Do only disable automagic when FSharp.Core is actually used

### LINKED ISSUE #3721: A new project fails to run
I created a completely new F# dotnet core project. when adding 
`<Import Project="..\..\.paket\Paket.Restore.targets" />`  and dotnet running it then it fails:

image

adding FSharp.Core made it work again. I wonder if it's related to https://github.com/fsprojects/Paket/pull/3706? /cc @atlemann @JohnyWS

### PR DESCRIPTION
**Pull request by forki**
fixes #3721

### CODE REVIEW

**Comment by forki** [comment]
@matthid then how about b8507bb?

**Comment by matthid** [comment]
I'm not a functional purist :) As long as it works. But yes thanks for taking care of this nit!

File: src/Paket.Core/Installation/RestoreProcess.fs [hunk 1]
```diff
@@ -324,18 +326,31 @@ let createPaketPropsFile (lockFile:LockFile) (cliTools:ResolvedPackage seq) (pac
...
+
     // When updating the PaketPropsVersion be sure to update the Paket.Restore.targets which checks this value
     let content =
         sprintf """<?xml version="1.0" encoding="utf-8" standalone="no"?>
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
     <PropertyGroup>
         <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
         <PaketPropsVersion>5.185.3</PaketPropsVersion>
-        <PaketPropsLoaded>true</PaketPropsLoaded>
+        <PaketPropsLoaded>true</PaketPropsLoaded>%s
```
Comment (atlemann) [inline src/Paket.Core/Installation/RestoreProcess.fs]: What's the %s at the end here?

Comment (forki) [reply src/Paket.Core/Installation/RestoreProcess.fs]: it's a big sprintf ".... %s ....".   I also the parameter coule of lines below

Comment (atlemann) [reply src/Paket.Core/Installation/RestoreProcess.fs]: Aah, guess context got lost using a cellphone to see the code...

**Pull request merged**